hapi/hedera-protobuf-java-api/src/main/proto/services/response_code.proto

```diff
@@ -1760,4 +1760,9 @@ enum ResponseCodeEnum {
      * which the network does not support.
      */
     GRPC_WEB_PROXY_NOT_SUPPORTED = 399;
+
+    /**
+     * An NFT transfers list referenced a token type other than NON_FUNGIBLE_UNIQUE.
+     */
+    NFT_TRANSFERS_ONLY_ALLOWED_FOR_NON_FUNGIBLE_UNIQUE = 400;
```

Comment:
Since this is a change to the specification, it needs to be in a HIP. Since this is a bug fix and not a feature, I think we can find an appropriate existing HIP to update rather than writing a whole new one.

Reply:
Ah yes [HIP-17 ("Non-fungible Tokens")](https://hips.hedera.com/hip/hip-17) would be perfect; I'll open a tiny PR adding this failure status there.